Products/ZenUI3/browser/resources/js/zenoss/form/LinkField.js

```diff
@@ -38,15 +38,21 @@ Ext.define("Zenoss.form.LinkField", {
             });
             value = items.join('<br/>');
         } else {
-            var linkMatch = value.match(/(?:<a href=")(.+)(?:">)/);
-            var nameMatch = value.match(/(?:<a href=.*">)(.+)(?:<\/a>)/);
-            if (linkMatch && nameMatch) {
-                var link = linkMatch[1];
-                var name = nameMatch[1];
-                value = Zenoss.render.link(null, link, name);
+            var div = Ext.get(document.createElement('div'));
```

Comment:
safely parse and extract html string to not get in troubles with regex with variety of urls and case when we have more than one <a> link in response.